tpl/js-templates.php

```diff
@@ -551,7 +551,8 @@ class="<?php echo esc_attr( implode( ' ', $classes ) ) ?>{{% if( rowColorLabel =
 		</div>
 
 		<div class="so-sidebar-tools">
-			<button class="live-editor-close button-primary"><?php esc_html_e('Done', 'siteorigin-panels') ?></button>
+			<button class="live-editor-save button-primary"><?php esc_html_e('Save', 'siteorigin-panels') ?></button>
```

Comment:
@gregpriday Perhaps this buttons label should be _Save Draft_ for new posts and _Update Post_ for existing posts?

Reply:
I'll do that fix soon.